docs/specification/appendix.md

```diff
@@ -118,4 +124,67 @@ The `ros2` profile describes how to create MCAP files for [ROS 2](https://docs.r
 
 #### Schema
 
-- `encoding`: MUST contain `ros2msg`
+- `encoding`: MUST contain either `ros2msg` or `ros2idl`
+
+## `gendeps`-style message definition delimiting
+
+For ROS1 and ROS2 message types, the message definition is stored in the [Schema record](./README.md#schema-op0x03) along with the definitions of any non-primitive field types. These definitions are concatenated together using a simple delimiting scheme:
+
+- The full content of the top-level message definition file is stored first.
+- Each dependent definition is preceded by:
+  - a line containing exactly 80 `=` characters, then
+  - A line containing only `MSG: <package resource name>` for that definition.
```

Comment:
for idl dependencies, do we expect the package resource name to be in the format `builtin_interfaces/msg/Time` or `builtin_interfaces/msg/Time.idl`? 

(would be good to specify this if it's not already covered somewhere)

Reply:
will document that we don't expect the file extension.